README.md

```diff
@@ -359,6 +359,32 @@ Some potential prompts:
 </div>
 
 <div class="cell markdown">
+    
+## Exercise 6
+
+</div>
+
+<div class="cell markdown">
+
+The AFLOW database is primarily built by decorating crystallographic prototypes, 
```

Comment:
Perhaps you could still explain a bit what a crystallographic prototype is. 
People who have not followed the lecture may also do the tutorial.